hi-ml-cpath/Makefile

```diff
@@ -92,7 +92,7 @@ define DEEPSMILEPANDATILES_ARGS
 endef
 
 define TCGACRCKSSLMIL_ARGS
---model=health_cpath.TcgaCrckSSLMIL
+--model=health_cpath.TcgaCrckSSLMIL --ssl_checkpoint=CRCK_SimCLR_1655731022_85790606
```

Comment:
Shouldn't this argument match the parameter name (`ssl_checkpoint_run_id`)?

Reply:
if we have some golden checkpoints what about we save them in a naming class and give them more telling names? and pass here the attribute of the class?